pkg/virtualKubelet/apiReflection/controller/outgoingReflectorsController.go

```diff
@@ -137,3 +138,10 @@ func (c *OutgoingReflectorsController) startNamespaceReflection(namespace string
 func (c *OutgoingReflectorsController) stopNamespaceReflection(namespace string) {
 	close(c.namespacedStops[namespace])
 }
+
+func (c *OutgoingReflectorsController) GetMirroringObject(api apimgmt.ApiType, namespace, name string) (interface{}, error) {
+	return c.apiReflectors[api].(ri.OutgoingAPIReflector).GetObjFromForeignCache(namespace, types.NamespacedName{
```

Comment:
you can simply use the `keyer` method of the `GenericApiReflector` instead of using a `types.NamespacedName`